configuration/messageproperties/messages_en.properties

```diff
@@ -3376,7 +3376,7 @@ ui.i18n.EncounterType.name.fac9d9a2-d0bc-11ea-9995-3c6aa7c392cc=PEDS
 
 #Viral Load History
 pih.app.hiv.viralLoadHistory.title=Viral load history
-pih.app.hiv.viralLoadHistory.date=Date
-pih.app.hiv.viralLoadHistory.qualitative=Status
+pih.app.date=Date
```

Comment:
You don't need to add pih.app.status or pih.app.date.  Use "pihcore.status" and "zl.date" for your labels.